src/cli_util.rs

```diff
@@ -1371,7 +1371,15 @@ fn serialize_config_value(value: config::Value) -> String {
 
 pub fn short_commit_description(commit: &Commit) -> String {
     let first_line = commit.description().split('\n').next().unwrap();
-    format!("{} ({})", short_commit_hash(commit.id()), first_line)
+    format!(
+        "{} ({})",
+        short_commit_hash(commit.id()),
+        if first_line.is_empty() {
+            "no description set"
+        } else {
+            first_line
+        }
+    )
 }
```

Comment:
We should eventually make this templated. I actually made that change for hg a few years ago.

Reply:
This function is actually only used to format commits inside commit description prompt files. It should be changed to use `write_commit_summary` with a plain formatter. That's the function that could be templated. In fact, I'll add a TODO.